client/components/payment-activity/test/index.test.tsx

```diff
@@ -129,9 +129,17 @@ describe( 'PaymentActivity component', () => {
 		// Check survey is rendered.
 		getByText( 'Are these metrics helpful?' );
 
-		// Check correct currency/value is displayed.
+		// Check TPV value is displayed with the correct currency symbol and format.
 		const tpvElement = getByLabelText( 'Total payment volume' );
-		expect( tpvElement ).toHaveTextContent( '€1.234,56' );
+		expect( tpvElement ).toHaveTextContent( /^€1.234,56/ );
```

Comment:
I opted for regex with the start-of-string `^` matcher to ensure the value isn't prefixed with `-`.

Reply:
This explanation of intent would make a great code comment, for future test-updaters :)

Reply:
Good idea → added in 2c4bd52dd54daefae5c2d9555ae20df4095a4f62